src/integrationTest/java/com/splunk/ibm/mq/integration/tests/JakartaPutGet.java

```diff
@@ -109,19 +110,21 @@ public static void runPutGet(
 
       // Create Jakarta objects
       context = cf.createContext();
-      logger.info("About to create queue {}", queueName);
       Destination destination = context.createQueue("queue:///" + queueName);
 
+      JMSConsumer consumer = context.createConsumer(destination);
+      consumer.setMessageListener(message -> {});
+
+      senderContext = cf.createContext();
+      Destination senderDestination = senderContext.createQueue("queue:///" + queueName);
```

Comment:
Instrumentation that creates queues huh? Users really want this? 🙈

Reply:
that's in the tests.